pandas/tests/series/test_analytics.py

```diff
@@ -1242,16 +1242,32 @@ def test_idxmin(self):
         result = s.idxmin()
         assert result == 1
 
-    def test_numpy_argmin(self):
-        # argmin is aliased to idxmin
-        data = np.random.randint(0, 11, size=10)
-        result = np.argmin(Series(data))
-        assert result == np.argmin(data)
+    def test_numpy_argmin_deprecated(self):
+        # See gh-16830
+        data = np.arange(10)
+
+        s = Series(data)
```

Comment:
I know this was like this before, but it would actually be good to use eg `index=np.arange(1, 11)`, just to have a distinction between positional and label based argmin

Reply:
Good call.

Reply:
I don't think you changed it correctly. It's the index of the Series that should be changed, not the actual `data`. Because now the tests should fail (the argmin should still be 0 (first location or first index label), while you changed it to assert to 1)